SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -693,3 +681,30 @@ FVector USpatialActorChannel::GetActorSpatialPosition(AActor* Actor)
 		return FVector::ZeroVector;
 	}
 }
+
+void USpatialActorChannel::SpatialViewTick()
+{
+	if (Actor != nullptr && !Actor->IsPendingKill() && IsReadyForReplication())
+	{
+		bool bOldNetOwned = bNetOwned;
+		bNetOwned = Actor->GetNetConnection() != nullptr;
```

Comment:
Pawns have no owners but can have changes in their controller, which changes their net connection, and there are also beacons I have not looked into too much. Also native Unreal does this check every tick and during my tests doing it on replication seemed to be flaky - but maybe it could work, just feels like a lot of exceptions.